docs/quick-start.md

```diff
@@ -211,6 +211,44 @@ collections:
     fields:
       - {label: "Language", name: "language"}
 ```
+
+### Data Files
```

Comment:
What you're describing here isn't actually specific to data files, it's actually about the `files` type collection (versus the `folder` type collection). Markdown files can work this way, too.

We should update this title accordingly, and have a separate section about the types of files we support (with examples). We should also move this out of quick start, I think it should all go in a new "Collections" page for now.